### PR TITLE
PW – Pricing updates

### PR DESCRIPTION
**Pull request by p-woznikowski**
## What

New pricing, new limits, new article

### CODE REVIEW

File: docs/guides/development/Limits.md [hunk 1]
```diff
@@ -10,107 +10,107 @@ order: 140
...
-| API requests per hour | Free plan: 100,<br>Startup plan: 1000,<br>Growth plan: 2000,<br>Professional plan: 5000 |
-| Bulk API requests per hour | Free plan: 1,<br>Startup plan: 10,<br>Growth plan: 20,<br>Professional plan: 50 |
-| Billing cycle API requests<br><br>Bulk API requests are not added to the billing cycle limit. | Free plan: 500,<br>Startup plan: 10,000,<br>Growth plan: 30,000,<br>Professional plan: 100,000 |
-| Client-side requests (e.g., validate) from a single IP address | 5 requests per 5 seconds |
+The following API limits apply to Business and Organization plans.
 
-To understand the basic limits of your account, check out [Pricing](https://www.voucherify.io/pricing) and [Fair Use Policy](https://www.voucherify.io/legal/fair-use-policy).
+| **Limit**                                                      | **Value**                                  |
+| :------------------------------------------------------------- | :----------------------------------------- |
+| API requests per minute                                        | Business: 100<br>Organization: 2000        |
```
Comment (karolinatrzcionka) [inline docs/guides/development/Limits.md:28]: formatowanie liczby, dajmy 2,000

File: docs/guides/development/Limits.md [hunk 2]
```diff
@@ -10,107 +10,107 @@ order: 140
...
+| Client-side requests (e.g., validate) from a single IP address | 5 requests per 5 seconds                   |
 
-## Notifications
+To understand the basic limits of your account, go to Voucherify [Pricing](https://www.voucherify.io/pricing "Voucherify Pricing") and [Fair Use Policy](https://www.voucherify.io/legal/fair-use-policy "Voucherify Fair Use Policy").
+
+## Enterprise Plan
 
-You can receive notifications in your app and through email when you use a certain percentage of your redemptions, API calls, or bulk API calls. The account owner can configure these thresholds and notifications in the `Project settings > API Usage` tab.
-To set this up, the account owner has two options:
+Users with an Enterprise plan have custom API request limits. Depending on the case, it can be counted per hour or per minute.
```
Comment (karolinatrzcionka) [inline docs/guides/development/Limits.md:36]: juz bedziemy mieli tylko per minute.

File: docs/guides/development/Limits.md [hunk 3]
```diff
@@ -10,107 +10,107 @@ order: 140
...
 
 Notifications for webhooks usage will be sent if the usage exceeds the limit for an hour, even though they are counted in daily periods. As a result, you will be informed in advance about any suspicious user activity.
 
-> 📘 What happens if you exceed your limits?
+> 📘 What Happens If Limits Are Exceeded?
 >
-> You can't exceed the number of API calls (both hourly and monthly) defined in your pricing plan. Exceeding this limit will block your access to the API immediately.
+> You cannot exceed the number of API calls (both hourly and monthly) defined in your pricing plan. Exceeding this limit will block your access to the API immediately.
+>
+> However, the Enterprise plan can use the automatic subscription upgrade that will upgrade their subscription plan if there is a risk of exceeding the limit. This ensures smooth service operation in case of sudden higher traffic. Contact [Voucherify Sales](https://www.voucherify.io/contact-sales) to learn more about the Enterprise plan.
```
Comment (karolinatrzcionka) [inline docs/guides/development/Limits.md:65]: tutaj mowimy o upgrade to the higher pricing tier, based on the pricing tiers defined in the Order Form.

File: docs/guides/development/Limits.md (hunk 1 above)
Comment (p-woznikowski) [reply docs/guides/development/Limits.md]: Poprawka w https://github.com/voucherifyio/voucherify-openapi/pull/840/commits/8c44a2f25a932ba49e1154ad8b7579f8427df958

File: docs/guides/development/Limits.md (hunk 2 above)
Comment (p-woznikowski) [reply docs/guides/development/Limits.md]: Poprawka w https://github.com/voucherifyio/voucherify-openapi/pull/840/commits/8c44a2f25a932ba49e1154ad8b7579f8427df958

File: docs/guides/development/Limits.md (hunk 3 above)
Comment (p-woznikowski) [reply docs/guides/development/Limits.md]: Poprawka w https://github.com/voucherifyio/voucherify-openapi/pull/840/commits/8c44a2f25a932ba49e1154ad8b7579f8427df958

**Pull request merged**